pulp_rpm/app/advisory.py

```diff
@@ -136,10 +138,20 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
             to_remove.append(previous_advisory_pk)
             to_exclude.append(added_advisory_pk)
         else:
-            raise AdvisoryConflict(_('Incoming and existing advisories have the same id and '
-                                     'timestamp but different and intersecting package lists. '
-                                     'At least one of them is wrong. '
-                                     f'Advisory id: {previous_advisory.id}'))
+            if previous_updated_version != added_updated_version:
```

Comment:
This should happen right after ` if previous_updated_date == added_updated_date:`

Reply:
```
if previous_updated_date != added_updated_date:
    if previous_pkglist.intersection(added_pkglist):
        if previous_updated_version < added_updated_version:
            ...
```